test/Microsoft.AspNetCore.Server.IISIntegration.Tests/IISMiddlewareTests.cs

```diff
@@ -79,14 +78,64 @@ public void UrlDelayRegistered()
                 .UseSetting("TOKEN", "TestToken")
                 .UseSetting("PORT", "12345")
                 .UseSetting("APPL_PATH", "/")
-                .UseIISIntegration();
+                .UseIISIntegration()
+                .Configure(app =>
+                {
+                    app.Run(context => Task.FromResult(0));
```

Comment:
Aesthetics. Otherwise you get this in the test output:
```
Application startup exception: System.InvalidOperationException: No service for type 'Microsoft.AspNetCore.Hosting.IStartup' has been registered.
   at Microsoft.Extensions.DependencyInjection.ServiceProviderServiceExtensions.GetRequiredService(IServiceProvider provider, Type serviceType)
   at Microsoft.Extensions.DependencyInjection.ServiceProviderServiceExtensions.GetRequiredService[T](IServiceProvider provider)
   at Microsoft.AspNetCore.Hosting.Internal.WebHost.EnsureStartup()
   at Microsoft.AspNetCore.Hosting.Internal.WebHost.EnsureApplicationServices()
   at Microsoft.AspNetCore.Hosting.Internal.WebHost.BuildApplication()
```